### PR TITLE
Monthly leaderboard

### PR DESCRIPTION
**Pull request by AtmegaBuzz**
Issue:  show monthly leaders on the leaderboard #827 
implemented filter by month leaderboard
image

### CODE REVIEW

**Review by fredfalcon — CHANGES_REQUESTED** [review]
We want the monthly leaderboard (that would show top winners for a particular  month and year) in addition to the global leaderboard- thanks!

**Comment by AtmegaBuzz** [comment]
@fredfalcon ok got it ill make seprate template for this UI.

**Comment by AtmegaBuzz** [comment]
@fredfalcon Here
image

File: website/templates/leaderboard.html [hunk 1]
```diff
@@ -46,33 +46,37 @@
...
-                <img src="{{ leader.socialaccount_set.all.0.get_avatar_url }}" class="profileimage">
-                {% else %}
-                <img src="{% gravatar_url leader.email 50 %}" class="profileimage">
-                {% endif %}
-                <a href="/profile/{{ leader.username }}">{{ leader.username }}</a>
-                <span class="pull-right badge">{{ leader.total_score }} Points</span>
-                {% if leader.userprofile.winnings %}
-                <span class="pull-right badge">${{ leader.userprofile.winnings|default:""|floatformat }}</span>
-                {% endif %}
+            {%if not leaderboard%}
```
Comment (fredfalcon) [inline website/templates/leaderboard.html:56]: This change is no longer needed

**Review by fredfalcon — CHANGES_REQUESTED** [review]
Please also add spaces to {%else%} tags

File: website/templates/leaderboard.html (hunk 1 above)
Comment (AtmegaBuzz) [reply website/templates/leaderboard.html:56]: @fredfalcon ive push the change for space in else tag

**Comment by AtmegaBuzz** [comment]
@fredfalcon i added spaces as you requested, are there more changes needed in this pr ?

**Review by fredfalcon — APPROVED** [review]
Where is the link to it?  Looks good

**Comment by AtmegaBuzz** [comment]
@fredfalcon the link is /leaderboard/monthly/?month=<month>&year=<year>

**Comment by fredfalcon** [comment]
@AtmegaBuzz how can a normal user get to it? Can you add a link from the normal leaderboard?

**Pull request merged**